nitro-protocol/readme.md

```diff
@@ -91,6 +91,6 @@ After deployment, run
 ETHERSCAN_API_KEY=<a-secret> INFURA_TOKEN=<another-secret> yarn hardhat --network rinkeby verify <DeployedContractAddress> 'ConstructorArgs'
 ```
 
-for each contract you wish to verify. Swap rinkeby for mainnet as appropriate.
+for each contract you wish to verify. Swap rinkeby for mainnet as appropriate!
```

Comment:
Is this:
- unrelated drive-by
- accidental
- intended to be "some changeset" so that the gh action runs
?